packages/core/react/src/useWallet.ts

```diff
@@ -37,7 +37,69 @@ export interface WalletContextState {
     signMessage: MessageSignerWalletAdapterProps['signMessage'] | undefined;
 }
 
-export const WalletContext = createContext<WalletContextState>({} as WalletContextState);
+const EMPTY_ARRAY: ReadonlyArray<never> = [];
+
+const DEFAULT_CONTEXT = {
+    autoConnect: false,
+    wallets: [],
+    wallet: null,
+    publicKey: null,
```

Comment:
You can remove these three, because you have defined getters below, right?

Reply:
True, but this insufficient overlap error kept showing up. I just fixed it by type casting `DEFAULT_CONTEXT` to `WalletContextState`.
<img width="566" alt="Screenshot 2022-03-04 at 2 48 43 PM" src="https://user-images.githubusercontent.com/39198645/156784758-5e4a4cf9-5214-4993-80a0-ad2687d70ce1.png">

Reply:
Yeah, makes sense. This might come back to bite us if we update the type of the context but _forget_ to update the implementation of the default context, but it's probably not the worst thing. If you're falling back to this default context you're already in a bad place.